db/controller/controller.js

```diff
@@ -27,8 +28,18 @@ exports.getArticlesById = (req, res, next) => {
     });
 };
 
+exports.getCommentsById = (req, res, next) => {
+  const id = req.params.article_id;
+  fetchCommentsById(id)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
This function is returning comments, so naming the returned value 'article' is confusing, please fix
Also it's best practise to send back a response in an object on a key, so {comments: <all the comments>}